### PR TITLE
fix: delete only excess data

### LINKED ISSUE #1091: scheduler job should wipe only unnecessary cells
It should delete only unnecessary data after batchUpdate.

### PR DESCRIPTION
**Pull request by tmatsuo**
fixes #1091

### CODE REVIEW

File: src/exportToSheets.ts [hunk 1]
```diff
@@ -95,4 +89,13 @@ export async function exportToSheets() {
       ],
     },
   });
+
+  // then clear the excess data
+  const start = values.length + 1;
+  const end = start + 10000;
```
Comment (bcoe) [inline src/exportToSheets.ts:95]: I would be tempted to pull the `10000` into a `const`, to explain what this magic value is doing.

File: src/exportToSheets.ts [hunk 2]
```diff
@@ -95,4 +89,13 @@ export async function exportToSheets() {
...
+
+  await sheets.spreadsheets.values.clear({
+    spreadsheetId,
+    range: `A${start}:Z${end}`,
```
Comment (bcoe) [inline src/exportToSheets.ts:99]: If I'm reading this correctly, we assume the new values will be at the head of the list, and the excess values will be any after the first N values.

Once concern is, are we sure that the insertion into sheets isn't eventually consistent?

**Review by bcoe — COMMENTED** [review]
Looks pretty reasonable to me, but I'm tired and want to be careful with this change. Perhaps see if Jeff C., would be willing to take a look next week too?

File: src/exportToSheets.ts (hunk 2 above)
Comment (tmatsuo) [reply src/exportToSheets.ts:99]: Yes, it batchUpdates with A1

File: src/exportToSheets.ts (hunk 1 above)
Comment (tmatsuo) [reply src/exportToSheets.ts:95]: ok

**Comment by tmatsuo** [comment]
@bcoe Thanks, PTAL

**Review by chingor13 — APPROVED** [review]

**Pull request merged**